### PR TITLE
Fixes #31551 - remove ovirt_use_v4 from oVirt CR attributes

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class RemoveOvirtUseApiFromAttr < ActiveRecord::Migration[6.0]
+  def change
+    compute_resources = ComputeResource.all.unscoped.where(type: 'Foreman::Model::Ovirt')
```
Comment (tbrisker) [inline db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb]: not important but you could also do:
```suggestion
    compute_resources = Foreman::Model::Ovirt.unscoped
```

File: db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    compute_resources.each do |ovirt_resource|
+      ovirt_resource.attrs = ovirt_resource.attrs.except(:ovirt_use_v4)
+      ovirt_resource.save!(validate: false)
+    rescue => e
+      logger.warn("could save compute attributes due to:#{e}")
```
Comment (tbrisker) [inline db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb]: This is the only important comment, others are just informative suggestions:
```suggestion
      logger.warn("could not migrate compute attributes for #{ovirt_resource.name} compute resource due to: #{e}")
```

File: db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class RemoveOvirtUseApiFromAttr < ActiveRecord::Migration[6.0]
+  def change
+    compute_resources = ComputeResource.all.unscoped.where(type: 'Foreman::Model::Ovirt')
+    compute_resources.each do |ovirt_resource|
+      ovirt_resource.attrs = ovirt_resource.attrs.except(:ovirt_use_v4)
```
Comment (tbrisker) [inline db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb]: not important but you could also do:
```suggestion
      ovirt_resource.attrs.except!(:ovirt_use_v4)
```

**Review by tbrisker — CHANGES_REQUESTED** [review]
Looks good, just one small change please, and a few non-important comments you can feel free to ignore.

File: db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb (hunk 2 above)
Comment (shiramax) [reply db/migrate/20201224095619_remove_ovirt_use_api_from_attr.rb]: done

**Review by tbrisker — DISMISSED** [review]
Thanks @shiramax !

**Review by tbrisker — APPROVED** [review]

**Pull request merged**